test.sh

```diff
@@ -20,7 +20,22 @@ echo == Test BigMap service.
 echo
 
 dfx canister install BigMapPutGet
+
 LOOP="(true)";
 while [ "$LOOP" == "(true)" ]; do
     LOOP=$(dfx canister call BigMapPutGet doNextCall)
 done
+
+dfx canister call BigMapPutGet getFullLog --output raw > BigMapPutGet.raw
+
+echo BEGIN BigMapPutGet.raw
+cat BigMapPutGet.raw
+echo END BigMapPutGet.raw
+
+echo BEGIN 'didc decode `cat BigMapPutGet.raw` > BigMapPutGet.log'
+didc decode `cat BigMapPutGet.raw` > BigMapPutGet.log
```

Comment:
```suggestion
didc decode `cat BigMapPutGet.raw` -d .dfx/local/canisters/BigMapPutGet/BigMapPutGet.did -m getFullLog > BigMapPutGet.log 
```
If you add these flags, it will resolve the field hashes back to its original names.